src/lib/middleware/bearer-token-middleware.ts

```diff
@@ -10,7 +10,7 @@ export const bearerTokenMiddleware = ({
 
     return (req: Request, _: Response, next: NextFunction) => {
         if (
-            req.path.startsWith('/api/signal-endpoint/') ||
+            req.path.includes('/signal-endpoint/') ||
```

Comment:
we could also use `const baseUriPath = config.server.baseUriPath || '';` that could be more precise. The config is available in the constructor so it should be doable, but I leave it up to you

Reply:
Something like this? https://github.com/Unleash/unleash/pull/6767/commits/b647c8e6e3e3b474e3f4985f5940f5e4765c8dc5